src/LibHdb++Cassandra.cpp

```diff
@@ -627,8 +644,11 @@ void HdbPPCassandra::extract_and_bind_bool(CassStatement* statement,
 	}
 	else
 	{
-		ERROR_MSG("extract_read failed for attribute "<< data->attr_name << endl);
 		cass_statement_bind_null(statement,param_index);
+		stringstream error_desc;
+		error_desc << "Failed to extract the attribute from the Tango EventData. Possible type mismatch?" << ends;
```

Comment:
Please add the attribute name in the error description here too and in every exception where the attribute name is available.